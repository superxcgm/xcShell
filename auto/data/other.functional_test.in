x
mkdir ~/cd_dir