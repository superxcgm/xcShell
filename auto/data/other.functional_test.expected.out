xcShell_functional_test > x
No such file or directory
xcShell_functional_test > cd ~/Desktop/
Desktop > cd -
xcShell_functional_test > 